.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: cimg/node:16.14.0
       - image: rethinkdb:2.4.0
-      - image: circleci/postgres:12.3
+      - image: postgres:12.3
```

Comment:
+1 We should probably just switch to [`cimg/postgres:12.9`](https://circleci.com/developer/images/image/cimg/postgres) instead

Reply:
We should use cimg/postgres, we recently moved the node image to cimg/node as well.

Reply:
Just FYI, we are actually at 12.10 on DO

Reply:
Gotcha, so would it make sense to sync all of these version specifications up to 12.10?

Reply:
Sounds good. I'll update the local docker setups to use `postgres/12.10` and this to use `cimg/postgres:12.9` until they merge this change: https://github.com/CircleCI-Public/cimg-postgres/pull/42